src/ruby/spec/spec_helper.rb

```diff
@@ -0,0 +1,97 @@
+# This file was generated by the `rspec --init` command. Conventionally, all
```

Comment:
I would like to quote these from [this post](blog.nicksieger.com/articles/2007/01/02/customizing-rspec/).
>Spec Helper
>
>Most usages of RSpec that I’ve seen in the wild use a “spec helper” (spec_helper.rb). This file, following the pattern of Rails’ test_helper.rb, minimally contains require statements to pull in the RSpec code and any supporting code for running specs. By requiring the spec helper via a path relative to your spec (usually with require File.dirname(__FILE__) + '/spec_helper' or similar), it also allows you the convenience of running your specs one at a time from anywhere (say, by launching from your editor) or with rake or spec. This file is where your shared helper methods will go, and where they’ll get registered to be pulled into the contexts.